core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/GroupByAggregator.java

```diff
@@ -237,7 +235,7 @@ public void prepareRun(BatchAggregatorContext context) throws Exception {
     context.record(fllOperations);
   }
 
-  private void validateConditionalFunctions(Schema outputSchema, List<GroupByConfig.FunctionInfo> aggregates,
+  private void GroupByAggregationDefinitionvalidateConditionalFunctions(Schema outputSchema, List<GroupByConfig.FunctionInfo> aggregates,
```

Comment:
Is this method not used anywhere?
nit: G is in uppercase

Reply:
I had no intention to change this